x-pack/plugins/serverless_search/tsconfig.json

```diff
@@ -38,5 +38,7 @@
     "@kbn/data-views-plugin",
     "@kbn/kibana-utils-plugin",
     "@kbn/index-management-plugin",
+    "@kbn/core-application-browser",
```

Comment:
These two shouldn't be necessary here if we remove ApplicationStart and HttpStart as props above.